src/main/resources/static/js/login.js

```diff
@@ -1,13 +1,110 @@
 
 jQuery(document).ready(function() {
-	
-//	document.getElementById("loginBtn").onclick = function() {		  
-//		
-//    	var iframe = document.getElementById("mainFrame");
-//    	iframe.src = "http://localhost:8080/subscriptionpage.html";
-//	}
-	
-
-	
+
+    // /Start ## Global AJAX Sender function ##################################
+    var AjaxHttpSender = function () {};
+
+    AjaxHttpSender.prototype.sendAjax = function (url, type, data, callback) {
+        $.ajax({
+            url : url,
+            type : type,
+            data : data,
```

Comment:
The way spring security works by default is to look for an Authorization header.
Attaching such a header would look like this:
beforeSend: function (xhr) {
    xhr.setRequestHeader ("Authorization", "Basic " + btoa(username + ":" + password));
},
btoa being a function that encodes the user info to base64.